mmv1/products/compute/RegionNetworkFirewallPolicy.yaml

```diff
@@ -38,6 +38,10 @@ examples:
     primary_resource_id: 'policy'
     vars:
       policy_name: 'tf-test-policy'
+  - name: 'region_network_firewall_policy_roce'
+    primary_resource_id: 'policy'
+    vars:
+      policy_name: 'tf-test-policy'
```

Comment:
The `tf-test` prefix is actually added for you as long as your name includes a `-`. So for example, this could be changed to `rnf-policy`.

Reply:
Oh also, this may need to be `min_version: beta`

Reply:
Done, thanks.